packages/coil-extension/src/content/services/ContentScript.ts

```diff
@@ -160,14 +160,26 @@ export class ContentScript {
         } else if (request.command === 'onFrameAllowedChanged') {
           this.onFrameAllowedChanged(request)
         } else if (request.command === 'tip') {
-          debug('sendTip event')
-          const detail: TipEvent['detail'] = {
+          debug('sendTip event', request.data)
+          const detail: MonetizationProgressEvent['detail'] = {
```

Comment:
![image](https://user-images.githubusercontent.com/525211/110428087-58be6d00-80db-11eb-86db-84fb7258d3d7.png)

This doesn't seem right

Reply:
https://github.com/coilhq/coil/pull/4490/files#r589991430